zerver/actions/users.py

```diff
@@ -554,7 +554,9 @@ def do_update_outgoing_webhook_service(
                 user_id=bot_profile.id,
                 services=[
                     dict(
-                        base_url=service.base_url, interface=service.interface, token=service.token
```

Comment:
Once again, we usually don't reformat anything that's irrelevant to the overall issue of this PR.

Reply:
Fixed